src/core/state/currentSettings/featureFlags.ts

```diff
@@ -36,6 +38,17 @@ export const featureFlagsStore = createStore<FeatureFlagsStore>(
     persist: {
       name: 'featureFlagsStore',
       version: 10,
+      merge(_persistedState, currentState) {
+        const persistedState = _persistedState as FeatureFlagsStore; // fair to assume no one is gonna mess with this in inspect element
+        return {
+          ...currentState,
+          ...persistedState,
+          featureFlags: {
+            ...currentState.featureFlags,
+            ...persistedState.featureFlags,
+          },
+        };
+      },
```

Comment:
we shouldn't need to update the persister version for every new flag, only breaking changes like renaming an existing flag